benchmarks/osb/requirements.txt

```diff
@@ -38,7 +38,7 @@ ijson==2.6.1
     # via opensearch-benchmark
 importlib-metadata==4.11.3
     # via jsonschema
-jinja2==3.1.3
+jinja2==2.11.3
```

Comment:
Does anyone know why this version downgrade is included?

Reply:
Reverted it manually.